src/components/products/new/color-picker.svelte

```diff
@@ -61,6 +62,11 @@
       />
     </Button>
     <Dropdown>
+      {#if value != null}
+        <Button on:click={() => { value = null; dispatch('change'); }}>
```

Comment:
And who is listening to the event?

Reply:
https://github.com/Innopoints/frontend/blob/master/src/components/products/new/variety-form.svelte#L57

Reply:
huh, because the event wasn't being dispatched previously, and I don't see it being dispatched anywhere else. Shouldn't it get dispatched on any color selection?

Reply:
It gets forwarded on color selection